src/devices/Sgp30/ChecksumFailedException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Iot.Device.Sgp30
+{
+    /// <summary>
+    /// Exception raised where an SGP30 Checksum Validation fails during a device read operation.
+    /// </summary>
+    public class ChecksumFailedException : Exception
```

Comment:
Is it really necessary to add a specific exception? Will a IOException works?
We recommend not to create new exception except if really needed. For anythin related to communication, we prefer a simple IOExcetion.
On what I've seen in the code, I'm not convienced that this bring a lot of value.